GoogleForCreators/web-stories-wp
# Show a success message when restoring a revision

## Context



## Summary

Show a basic snackbar message on restoring a revision. 

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions

https://user-images.githubusercontent.com/237508/203111694-e4d251fd-3936-42b2-b3d2-a052fefc5a0d.mov




This PR can be tested by following these steps:

1. Create a story. 
2. Publish a story. 
3. Make an edit. 
4. Update story. 
5. Go to document tab. 
6. Click revisions. 
7. Change story and restore story. 
8. See message. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #12618
